### DISCUSSION TITLE
serde::Serialize is not implemented for entity::table::ActiveModel

### DISCUSSION THREAD

**Discussion by Ahmard** [SeaQL/sea-orm#1688] [Q&A]
Hello, please I want to use an instance of ActiveModel as actix json response and I keep getting 
`serde::Serialize is not implemented for entity::table::ActiveModel` error,
Unfortunately I can't derive Serialise since ActiveModel(s) are auto-generated.

**Comment by billy1624** [comment]
Hey @Ahmard, I think you can reuse the `Model` for Actix response. And you can update the corresponding `ActiveModel` with JSON value https://www.sea-ql.org/SeaORM/docs/basic-crud/insert/#set-activemodel-from-json-value
